inst/templates/tidy_contributing.md

```diff
@@ -0,0 +1,192 @@
+# Contributing to {{{package}}}
+
+
+## What is this?
+
+A `CONTRIBUTING.md` document (such as this one) has instructions for
```

Comment:
This intro seems a bit long for me. Could we consolidate into a single paragraph?

Reply:
@jennybc I totally tried to do that — I literally copied your frontmatter from contributing.md for googledrive, but it's not showing up for some reason... Will investigate further

Reply:
I'm getting it because I have CONRIBUTING.Rmd and this in my frontmatter:

https://github.com/tidyverse/googledrive/blame/61f0047474ba4bee37b334fd0e0f3c4a9807a7a0/CONTRIBUTING.Rmd#L5-L6

But I'm not sure this is a good justification to use Rmd here. Probably better to just do it by hand.

Reply:
Yeah, I think it's better to sticking with plain md.

Reply:
Added TOC to md